src/Jackalope/Node.php

```diff
@@ -1522,6 +1522,7 @@ public function getIterator()
      * @param mixed   $value
      * @param string  $type
      * @param boolean $internal whether we are setting this node through api or internally
+     * @param boolean $validate whether we are validating the change or not
```

Comment:
lets add that this may only be set to false by other parts of the jackalope library.